scripts/install/jellyfin.sh

```diff
@@ -1,33 +1,20 @@
 #!/usr/bin/env bash
 #
-# authors: liara userdocs
+# authors: liara userdocs flying-sausages
 #
 # Licensed under GNU General Public License v3.0 GPL-3 (in short)
 #
 ########
 ######## Variables Start
 ########
 #
-# Get our main user credentials.
-username="$(cat /root/.master.info | cut -d: -f1)"
-password="$(cat /root/.master.info | cut -d: -f2)"
-#
-# This will generate random ports for the script to use with applications between the range 10001 to 32001.
-app_port_http="$(shuf -i 10001-32001 -n 1)" && while [[ "$(ss -ln | grep -co ''"${app_port_http}"'')" -ge "1" ]]; do app_port_http="$(shuf -i 10001-32001 -n 1)"; done
-app_port_https="$(shuf -i 10001-32001 -n 1)" && while [[ "$(ss -ln | grep -co ''"${app_port_https}"'')" -ge "1" ]]; do app_port_https="$(shuf -i 10001-32001 -n 1)"; done
-echo_log_only "app_port_http = $app_port_http, app_port_https = $app_port_https"
-#
-# Get our exertnal IP 4 address and set it as a variable.
-ip_address="$(curl -s4 icanhazip.com)"
-#
-# Define the location where we want the application installed
-install_dir="/opt/jellyfin"
-#
-# Define the location where we want ffmpeg installed.
-install_ffmpeg="/opt/ffmpeg"
+# Source the global functions we require for this script.
+. /etc/swizzin/sources/functions/utils
+. /etc/swizzin/sources/functions/ssl
 #
-# Set an installation temporay directory.
-install_tmp="/tmp/jellyfin"
+# Get our some useful information from functions in the sourced utils script
+username="$(_get_master_username)"
+dist_info # get our distribution ID, set to DIST_ID, and VERSION_CODENAME, set to DIST_CODENAME, from /etc/os-release
```

Comment:
I'd do this in a separate PR where it's applied anywhere necessary otherwise this won't be implemented right I think. Also might need a separate focus and review. I'm not the one to talk with my 150+ file PRs hahaha so let me know if you think it makes sense

Reply:
i just moved it to the script. if the change gets added, it can be removed.